pageserver/src/tenant/timeline.rs

```diff
@@ -189,6 +190,14 @@ pub struct Timeline {
     state: watch::Sender<TimelineState>,
 }
 
+#[derive(Debug, PartialEq, Eq, Clone, Copy)]
+enum InitialLogicalSizeComputationState {
+    NotStarted,
+    Running,
+    FailedWillRetryNextTime,
```

Comment:
I guess some of the earlier ondemand work added additional failure modes to the size calculation that at least I wasn't aware of. Do you have any pointers on what those are?

Reply:
The new failure mode is that some `self.get_download()` call will fail due to network error or whatever.
Previously, we only accessed the local filesystem, so, failure was quite unlikely.

Reply:
But there won't be an additional enum variant here if that's what you're asking.

Reply:
> But there won't be an additional enum variant here if that's what you're asking.

Aah no, wasn't asking for a variant. Thanks for the pointer.